pkg/snapstore/abs_snapstore_test.go

```diff
@@ -19,12 +19,14 @@ import (
 	"github.com/Azure/azure-sdk-for-go/sdk/storage/azblob/blockblob"
 	"github.com/Azure/azure-sdk-for-go/sdk/storage/azblob/container"
 	"github.com/gardener/etcd-backup-restore/pkg/snapstore"
+	"k8s.io/utils/pointer"
```

Comment:
I'm removing this package throughout the code base in an upcoming PR which is upgrading a lot of dependencies.

Will let this be like this for now since I'll be changing things in an upcoming PR anyway. Does that work?

Reply:
> I'll be changing things in an upcoming PR

Yeah that works. Thanks